### PR TITLE
Student can see the grades of a module divided  by week, and category

### PR DESCRIPTION
**Pull request by cdsandoval**
Close: #51 #49

### CODE REVIEW

File: api/db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_07_24_155639) do
+<<<<<<< HEAD
```
Comment (luciagirasoles) [inline api/db/schema.rb]: I think you need to fix this conflict

File: api/db/seeds.rb [hunk 2]
```diff
@@ -1,63 +1,107 @@
 # Create users
 User.destroy_all
 user1 = User.create(name: 'Ricardo Yrupailla', email: 'ry.yrupailla@gmail.com', password: '123456')
+<<<<<<< HEAD
 user2 = User.create(name: 'Mayra Navarro', email: 'mnavarro@able.co', password: '123456')
 user3 = User.create(name: 'Mayra Navarro', email: 'luciagirasoles+1@gmail.com', password: '123456')
 user4 = User.create(name: 'carlos Sandoval', email: 'csandoval@able.co', password: '123456')
+=======
+User.create(name: 'Mayra Navarro', email: 'mnavarro@able.co', password: '123456')
```
Comment (luciagirasoles) [inline api/db/seeds.rb]: I think you need to resolve these

Comment (cdsandoval) [reply api/db/seeds.rb]: Yes, sure. Did not notice that. Going to fix it.

File: api/db/schema.rb (hunk 1 above)
Comment (cdsandoval) [reply api/db/schema.rb]: Yes, sure. Did not notice that. Going to fix it.

File: client/src/views/Login.jsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ import HomeLayout from "../components/layouts/HomeLayout";
 function Login() {
   const login = useLogin();
   const user = useUser();
-  const [email, setEmail] = React.useState("mnavarro@able.co");
+  const [email, setEmail] = React.useState("ry.yrupailla@gmail.com");
```
Comment (luciagirasoles) [inline client/src/views/Login.jsx]: why did you change my name? :(

File: api/spec/controllers/modulo_grades_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe ModuloGradesController, type: :controller do
+
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
```
Comment (byteconde) [inline api/spec/controllers/modulo_grades_controller_spec.rb]: An alternative way to specify the status is using HTTP CODES 
Link: https://www.restapitutorial.com/httpstatuscodes.html

Comment (luciagirasoles) [reply api/spec/controllers/modulo_grades_controller_spec.rb]: I prefer this way instead of http codes, this is more descriptive
